gallery/visualtests/nxCollapsibleItems.js

```diff
@@ -106,6 +107,30 @@ describe('NxCollapsibleItems', function() {
     it('looks right', simpleTest(disabledTreeViewSelector));
   });
 
+  describe('NxCollapsibleItems with action content', function() {
+    it('looks right collapsed', simpleTest(actionContentCollapsibleItemsSelector));
+    it('looks right expanded', async function() {
+      const firstTreeSelector =
+        `${actionContentCollapsibleItemsSelector} .nx-collapsible-items:first-child .nx-collapsible-items__trigger`;
+      const secondTreeSelector =
+        `${actionContentCollapsibleItemsSelector} .nx-collapsible-items:nth-child(2) .nx-collapsible-items__trigger`;
+
+      const iconDropdownToggleSelector =
+        `${actionContentCollapsibleItemsSelector} .nx-collapsible-items:first-child .nx-icon-dropdown__toggle`;
+      const [firstTree, secondTree, iconDropdownToggle] =
+        await waitAndGetElements(firstTreeSelector, secondTreeSelector, iconDropdownToggleSelector);
+
+      await firstTree.click();
+      await secondTree.click();
+
+      await wait(350);
+
+      await iconDropdownToggle.click();
+
+      await simpleTest(actionContentCollapsibleItemsSelector)();
```

Comment:
I'd actually rather test this with the first collapse section closed, in order to prove that the dropdown menu doesn't get cut off when it overflows out the bottom of the `NxCollapsibleItems`

Reply:
> I'd actually rather test this with the first collapse section closed, in order to prove that the dropdown menu doesn't get cut off when it overflows out the bottom of the NxCollapsibleItems

At the same time though, I think it's important to test that the two click targets don't interfere with each other: that clicking the collapse trigger doesn't open or close the dropdown, and that clicking the action button doesn't collapse or expand the items.  That should probably just be a separate test.

Reply:
> > I'd actually rather test this with the first collapse section closed, in order to prove that the dropdown menu doesn't get cut off when it overflows out the bottom of the NxCollapsibleItems
> 
> At the same time though, I think it's important to test that the two click targets don't interfere with each other: that clicking the collapse trigger doesn't open or close the dropdown, and that clicking the action button doesn't collapse or expand the items. That should probably just be a separate test.

Thanks for the feedback! I updated the tests here: 6366984bd3762ab77d0ab0c90fda012da1e53315